yarn-project/circuits.js/src/hints/build_hints.test.ts

```diff
@@ -113,11 +111,6 @@ describe('buildNullifierReadRequestHints', () => {
     const hints = await buildHints();
     expect(hints).toEqual(expectedHints);
   });
-
-  it('throws if reading an unknown nullifier', async () => {
```

Comment:
Remove this test because the hints builder can assume the oracle will always return a value. If everything is processed correctly, it shouldn't be requesting for an unknown nullifier.
However, the oracle will still throw if the requested nullifier does not exists.